### PR TITLE
Interaction - Mission: Cauterize, Uncertain Destinations

### PR DESCRIPTION
**Pull request by claywar**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?
Implements RoV Mission: Cauterize

## Steps to test these changes
Follow guides for this mission, along with notes contained within the mission script.

### CODE REVIEW

File: sql/npc_list.sql [hunk 1]
```diff
@@ -12849,7 +12849,7 @@ INSERT INTO `npc_list` VALUES (17207824,'qm4','???',102,426.315,8.563,-163.337,1
 INSERT INTO `npc_list` VALUES (17207825,'Monberaux','Monberaux',0,0.000,0.000,0.000,0,40,40,0,1,0,6,27,0x0000260300000000000000000000000000000000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17207826,'Wolfgang','Wolfgang',0,0.000,0.000,0.000,0,40,40,0,1,0,6,27,0x0100020114101920193019401950006000700000,32,NULL,1);
 -- airship
--- INSERT INTO `npc_list` VALUES (17207827,'','',0,0.000,0.000,0.000,0,40,40,19,0,0,2,3,0x040000000B000000000000000000000000000000,32,null,1);
+INSERT INTO `npc_list` VALUES (17207827,x'0B','NPC',0,0.000,200.000,0.000,65537,50,50,19,0,0,0,3,0x0400000000000000000000000000000000000000,0,NULL,1);
```
Comment (neuromancerxi) [inline sql/npc_list.sql:12852]: Stray x outside of single quotes.

Comment (WinterSolstice8) [reply sql/npc_list.sql:12852]: That's intentional, x'0B' = character value of 0x0B, it's used for non-printable NPC names, see https://github.com/LandSandBoat/server/pull/2483 for a more obvious example of what this means

Comment (neuromancerxi) [reply sql/npc_list.sql:12852]: Thanks for the clarification, first time I've come across it so it stood out as a potential typo to me.

Comment (WinterSolstice8) [reply sql/npc_list.sql:12852]: Oh yeah, it certainly is weird isn't it? It took me a good while to figure out how to use non-printable characters in the db. Very counter-intuitive compared to most other languages that lets you escape in string such as "\x0B"

File: scripts/zones/Batallia_Downs/IDs.lua [hunk 2]
```diff
@@ -21,6 +21,7 @@ zones[xi.zone.BATALLIA_DOWNS] =
         CARRIED_OVER_POINTS      = 7023,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7024,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
Comment (WinterSolstice8) [inline scripts/zones/Batallia_Downs/IDs.lua:24]: "ou" are unable

File: scripts/zones/Batallia_Downs_[S]/IDs.lua [hunk 3]
```diff
@@ -18,6 +18,8 @@ zones[xi.zone.BATALLIA_DOWNS_S] =
         CARRIED_OVER_POINTS     = 7001, -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER            = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS      = 7047, -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
Comment (WinterSolstice8) [inline scripts/zones/Batallia_Downs_[S]/IDs.lua:21]: "ou" are unable

File: scripts/zones/Rolanberry_Fields/IDs.lua [hunk 4]
```diff
@@ -20,6 +20,7 @@ zones[xi.zone.ROLANBERRY_FIELDS] =
         CARRIED_OVER_POINTS      = 7023,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7024,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
Comment (WinterSolstice8) [inline scripts/zones/Rolanberry_Fields/IDs.lua:23]: "ou" are unable

File: scripts/zones/Rolanberry_Fields_[S]/IDs.lua [hunk 5]
```diff
@@ -17,6 +17,8 @@ zones[xi.zone.ROLANBERRY_FIELDS_S] =
         CARRIED_OVER_POINTS     = 7001, -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER            = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS      = 7047, -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
Comment (WinterSolstice8) [inline scripts/zones/Rolanberry_Fields_[S]/IDs.lua:20]: "ou" are unable

File: scripts/zones/Sauromugue_Champaign/IDs.lua [hunk 6]
```diff
@@ -20,6 +20,7 @@ zones[xi.zone.SAUROMUGUE_CHAMPAIGN] =
         CARRIED_OVER_POINTS      = 7023,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7024,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7025,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7069,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
Comment (WinterSolstice8) [inline scripts/zones/Sauromugue_Champaign/IDs.lua:23]: "ou" are unable

File: scripts/zones/Sauromugue_Champaign_[S]/IDs.lua [hunk 7]
```diff
@@ -17,6 +17,8 @@ zones[xi.zone.SAUROMUGUE_CHAMPAIGN_S] =
         CARRIED_OVER_POINTS      = 7001,  -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY  = 7002,  -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER             = 7003,  -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
+        UNABLE_TO_PROGRESS       = 7047,  -- ou are unable to make further progress in Rhapsodies of Vana'diel due to an event occurring in the [Chains of Promathia/Treasures of Aht Urhgan/Wings of the Goddess/Seekers of Adoulin/Rise of the Zilart] missions.
```
Comment (WinterSolstice8) [inline scripts/zones/Sauromugue_Champaign_[S]/IDs.lua:20]: "ou" are unable

**Review by WinterSolstice8 — CHANGES_REQUESTED** [review]
Just a few missing "Y"s... 👍

File: scripts/zones/Batallia_Downs/IDs.lua (hunk 2 above)
Comment (claywar) [reply scripts/zones/Batallia_Downs/IDs.lua:24]: See: https://github.com/zach2good/UpdateExtractor/tree/main/out
This is intentional.

**Review by WinterSolstice8 — APPROVED** [review]
missing "Y" is from some bit magic corrupting the first character, so LGTM in that case

**Pull request merged**